lib/webhook.ts

```diff
@@ -0,0 +1,7 @@
+export interface Webhook {
```

Comment:
I'm thinking about maybe using Prisma generated types here; it has a lot of overlap. 

```javascript
model Webhook {
  id            Int       @id @default(autoincrement())
  uid           String    @unique
  userId        Int       
  subscriberUrl String
  eventType     WebhookEventTypes[]
  createdAt     DateTime  @default(now())
  active        Boolean   @default(true)         
  eventTriggers WebhookTriggerEvents[]
}

import { Webhook as PrismaWebhook } from "@prisma/client";
export type Webhook = PrismaWebhook & { prevState: null; };
// sidenote, prevState does not seem to be used anywhere?
```

Reply:
It does have a lot of overlap but for the eventType field.